src/Slider.js

```diff
@@ -0,0 +1,235 @@
+import React, { PureComponent } from "react";
+import { View, Animated, StyleSheet, PanResponder } from "react-native";
+import PropTypes from "prop-types";
+import GalioTheme, { withGalio } from './theme';
+
+
+class Slider extends PureComponent {
+  constructor(props) {
+    super(props);
+    this.state = {
+      containerSize: { width: 0, height: 0 },
+      trackSize: { width: 0, height: 0 },
+      thumbSize: { width: 0, height: 0 },
+      measured: false, //hide the UI until we measure the View
+    };
+
+    this.position = new Animated.Value(this.props.value); //recieve value from user
```

Comment:
You can move it outside the constructor

Reply:
Because you're using **this.props.value** into the constructor which is an anti-pattern
- **super** is accessing the parent props and make them available as **props**

Reply:
I'm not using this.props.value anymore.